docs/docs/topics/schema.mdx

```diff
@@ -91,11 +91,110 @@ The `kind` of a model is generated by concatenating the `namespace` and the `nam
 
 #### Relationship kinds
 
-- `Generic`: Default relationship without specific significance
-- `Attribute`: Relationship of type Attribute are represented in the detailed view and the list view
-- `Component`: Indicate a relationship with another node that is a component of the current node. Example: Interface is a component to a Device
-- `Parent`: Indicate a relationship with another node that is a parent to the current node. Example: Device is a parent to an Interface
-- `Group`: Indicate a relationship to a member or a subscriber of a group
+- `Generic`: A flexible relationship that has no specific functional significance. It is commonly used when an entity doesn't fit into more specialized categories like `Component` or `Parent`.
+- `Attribute`: A relationship where related entities' attributes appear directly in the detailed view and can influence list views. It's used for linking key information, like statuses or roles.
+- `Component`: This relationship indicates that one entity is a part of another and typically appears in a separate tab. It represents a composition-like relationship where one node is a component of the current node.
+- `Parent`: This relationship defines a hierarchical link, with the parent entity often serving as a container or owner of another node. **Parent relationships are mandatory** and allow filtering in the UI, such as showing all children for a given parent.
```

Comment:
showing all children > showing all components
maybe we should highlight that component / parent relationships typically belong together